src/components/ItemFilters/ItemFilters.tsx

```diff
@@ -160,7 +131,55 @@ const FiltersContainer = ({
             onSubmit={handleYearSubmit}
           />
         </Box>
-      </Box>
+      </>
+    )
+  }
+
+  // function passed to filterElements map callback for generating renderMultiSelect prop of MultiSelectGroup
+  const renderMultiSelect = (
+    checkboxGroup: FilterCheckboxGroup,
+    appliedFilterData = {}
+  ) => {
+    return (
+      <MultiSelect
+        buttonText={checkboxGroup.name}
+        id={`${checkboxGroup.id}-multi-select`}
+        data-testid={`${checkboxGroup.id}-multi-select`}
+        items={checkboxGroup.items}
+        key={checkboxGroup.id}
+        width="fitContent"
+        __css={{ flex: 1 }}
```

Comment:
You shouldn't need to pass these. What's this doing?

Reply:
I added that when I was getting some weird styling before I made the other fixes you suggested to FilterBarInline's props. Removed.